index.js

```diff
@@ -1,28 +1,30 @@
 module.exports = {
-    parserOptions: {
-        ecmaVersion: 11,
-        sourceType: 'module'
-    },
     extends: [
         'airbnb',
         'plugin:promise/recommended',
-
     ],
     plugins: [
         'react-hooks'
     ],
     rules: {
         'import/order': ['error', {
-            'groups': ['builtin', 'external', 'internal', 'parent', 'sibling', 'index']
+            groups: ['builtin', 'external', 'internal', 'parent', 'sibling', 'index']
+        }],
+        'max-len': ['error', {
+          code: 120,
+          ignoreTrailingComments: true,
+          ignoreUrls: true,
+          ignoreStrings: true,
+          ignoreTemplateLiterals: true,
+          ignoreRegExpLiterals: true,
         }],
         'object-curly-newline': ['error', {
-            'multiline': true,
-            'minProperties': 2,
-            'consistent': true
+            multiline: true,
+            minProperties: 2,
+            consistent: true
         }],
         'object-curly-spacing': ['error', 'always'],
-        'object-property-newline': ['error', { 'allowMultiplePropertiesPerLine': false }],
```

Comment:
`object-property-newline` was IMHO useless as the functionality is covered by `object-curly-newline`